src/common/services/elrond-communication/elrond-tools.service.ts

```diff
@@ -0,0 +1,129 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { elrondConfig } from 'src/config';
+import axios, { AxiosRequestConfig } from 'axios';
+import { ApiConfigService } from 'src/utils/api.config.service';
+import * as Agent from 'agentkeepalive';
+import { PerformanceProfiler } from '@elrondnetwork/erdnest';
+import { MetricsCollector } from 'src/modules/metrics/metrics.collector';
+import { NativeAuthSigner } from '@elrondnetwork/erdnest/lib/src/utils/native.auth.signer';
+import BigNumber from 'bignumber.js';
+
+@Injectable()
```

Comment:
maybe use the api.service it already has all axios configs

Reply:
Done